protostar/commands/test/fuzzing/strategies/__init__.py

```diff
@@ -0,0 +1,24 @@
+from types import SimpleNamespace
+from typing import Any
+
+from protostar.starknet.hint_local import HintLocal
+from .felt import (
+    IntegersStrategyDescriptor,
+    SignedFeltStrategyDescriptor,
+    UnsignedFeltStrategyDescriptor,
+)
+
+namespace = SimpleNamespace(
+    integers=IntegersStrategyDescriptor,
+    signed=SignedFeltStrategyDescriptor,
+    unsigned=UnsignedFeltStrategyDescriptor,
+)
+
+
+class StrategiesHintLocal(HintLocal):
+    @property
+    def name(self) -> str:
+        return "st"
```

Comment:
Is it worth to sacrifice readability?

Reply:
I found writing `strategies` everywhere cumbersome and actually Hypothesis suggests using the `st` alias in their docs. But I'm open to renaming this. @MaksymilianDemitraszek wdyt?

Reply:
I don't have a clear opinion tbh. I have a feeling that `st` is a bit too cryptic. You can't look at code and have an intuition what this variable may be about.

Maybe it would be better to use a full name and let users alias it by themselves if they need to? using name shortcuts works in python because you can always check what it is in imports, here users can't do that

Reply:
We can consider introducing cheatcode aliases in the future so users can use both `st` and `strategies`

Reply:
renamed

Reply:
@MaksymilianDemitraszek I would prefer not to offer built-in aliases because that would lead to having people do different conventions and fight